sql/upgrade/sl3.0.sql

```diff
@@ -903,60 +910,71 @@ ALTER TABLE gl ENABLE TRIGGER gl_audit_trail;
 
 ALTER TABLE ar DISABLE TRIGGER ar_audit_trail;
 
---TODO: Handle amount_tc and netamount_tc
 insert into ar
-(entity_credit_account, person_id,
+        (entity_credit_account, person_id,
         id, invnumber, transdate, taxincluded,
         amount_bc, netamount_bc,
         amount_tc, netamount_tc,
 <?lsmb IF VERSION_COMPARE(lsmbversion,'1.6') < 0; ?>
         paid, datepaid,
 <?lsmb END; ?>
-        duedate, invoice, ordnumber, curr, notes, quonumber, intnotes,
+        duedate, invoice, ordnumber,
+        curr,
+        notes, quonumber, intnotes,
         shipvia, language_code, ponumber, shippingpoint,
-        on_hold, approved, reverse, terms, description)
+        on_hold, approved, reverse,
+        terms, description)
 SELECT
         customer.credit_id,
         (select entity_id from :slschema.employee WHERE id = ar.employee_id),
-        ar.id, invnumber, transdate, ar.taxincluded, amount, netamount,
-        CASE WHEN exchangerate IS NOT NULL THEN amount/exchangerate ELSE 0 END,
-        CASE WHEN exchangerate IS NOT NULL THEN netamount/exchangerate ELSE 0 END,
+        ar.id, invnumber, transdate, ar.taxincluded,
+        amount, netamount,
+        CASE WHEN exchangerate IS NOT NULL AND exchangerate <> 1 THEN amount/exchangerate ELSE 0 END,
+        CASE WHEN exchangerate IS NOT NULL AND exchangerate <> 1 THEN netamount/exchangerate ELSE 0 END,
 <?lsmb IF VERSION_COMPARE(lsmbversion,'1.6') < 0; ?>
         paid, datepaid,
 <?lsmb END; ?>
```

Comment:
oh. The conditional *is* there. The question about different versions of schemas still applies though.